pheps/phep-9999.md

```diff
@@ -0,0 +1,162 @@
+```
+PHEP: 9999
+Title: PyHC Package Tiering
+Author: Julie Barnum <julie.barnum@lasp.colorado.edu> <https://orcid.org/0000-0001-8755-0694>
+Discussions-To: https://github.com/heliophysicsPy/standards/pull/25
+Revision: 1
+Status: Draft
+Type: Process
+Content-Type: text/markdown; charset=UTF-8; variant=CommonMark
+Created: 
+Post-History: 09-July-2024 
+```
+
+# Abstract
+<a name="abstract"></a>
+This PHEP establishes a new tiering structure to PyHC projects, which will automatically affect PyHC packages once it goes into effect. Included herein is information on requirements for each of the new four tiers of PyHC projects (Gold, Silver, Bronze, and Honorable Mention), as well as benefits accrued at each tier. 
```

Comment:
I believe "Honorable Mention" should be renamed to "Copper" here.

Reply:
Good catch, thanks!